mm2src/mm2_bitcoin/keys/Cargo.toml

```diff
@@ -9,5 +9,7 @@ base58 = "0.1"
 bech32 = "0.8.0"
 libsecp256k1 = "0.2.2"
 bitcrypto = { path = "../crypto" }
+gstuff = { version = "0.6", features = ["nightly"] }
```

Comment:
Our long-term goal is to stop using `ERR/ERRL/try_s` macros. Please remove this dependency. Adding a common crate to use MmError here is not mandatory now, it's fine to return a custom error type wherever required in underlying deps.